Testing/Functional/Products/Tests/TestData/sharepoint-commercial-data.pson

```diff
@@ -0,0 +1,8 @@
+@{
+    TenantDomain = 'y2zj1.onmicrosoft.com'
+    TenantDisplayName = 'y2zj1'
+    AppId = 'db791fde-1ff0-493f-be3d-03d9ba6cb087'
```

Comment:
It is my opinion that this info should be in GitHub as a secret similar to Smoke Test.

Reply:
Already done, thank you.